src/factories/Container.ts

```diff
@@ -49,10 +51,10 @@ module n3Charts.Factory {
       var {left, top} = event.currentTarget.getBoundingClientRect();
 
       var xScale = this.factoryMgr.get('x-axis').scale;
-      var x = xScale.invert(event.x - left - dim.margin.left);
+      var x = xScale.invert(event.clientX - left - dim.margin.left);
```

Comment:
I've checked in both FF and Chrome, yeah. You can check with multiple charts by running `gulp demo` and then going to localhost:1234/test/e2e/ !